README.md

```diff
@@ -3,7 +3,7 @@
 React Native module to get launch arguments. Make passing parameters from testing tool to react native super easy.
 
 Mostly it's made for using
-* [`launchArgs parameter of device.launchApp method`](https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-launchargsadditional-process-launch-arguments) of [Detox](https://github.com/wix/Detox/)
+* [`launchArgs parameter of device.launchApp method`]([https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-launchargsadditional-process-launch-arguments](https://wix.github.io/Detox/docs/api/device/#deviceapplaunchargs)) of [Detox](https://github.com/wix/Detox/)
```

Comment:
not sure how link inside link should work. maybe this:

```suggestion
* [`launchArgs parameter of device.launchApp method`](https://wix.github.io/Detox/docs/api/device/#deviceapplaunchargs) of [Detox](https://github.com/wix/Detox/)
```

Reply:
fixed